rfcs/0130-stalled-rfcs.md

```diff
@@ -0,0 +1,145 @@
+    Merged
+    style Merged fill:#080,color:#FFF
+    
+    Rejected[Closed - Rejected]:::closed
+
+    Withdrawn[Closed - Withdrawn]:::closed
+    Discuss & Draft --->|Author Withdraws| Withdrawn
+
+    classDef closed fill:#800,color:#FFF
+```
+
+There are 3 new transitions in this diagram.
+
+## On Hold
+
+This is for cases where the author is unable or unwilling to update the RFC in a timely fashon and would like to indicate that the RFC should not continue to be pushed forward for the time being.
+
+The author can [mark the PR as a Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft) and the existing label [status: on hold](https://github.com/NixOS/rfcs/pulls?q=is%3Apr+label%3A%22status%3A+on+hold%22+is%3Aclosed) can be applied to the PR by the NixOS RFC Steering Committee for tracking reasons.
+
+RFCs in this status will be completely ignored by the NixOS RFC Steering Committee. It is also expected that community members will largely refrain from review until the author has indicated that the RFC is again ready for review.
+
+## Can't Find Shepherds
+
+If an RFC is unable to find adaqite number of shepherds in 2 months it will be considered "low interest". The RFC will be closed to indicate that there is no expection of continuous forward progress.
+
+After 1 month of being open the following message will be added to the PR to suggest ways to find more support:
+
+> This RFC has not acquired enough shepherds. This typically shows lack of interest from the community. In order to progress [a full shepherd team is required](https://github.com/NixOS/rfcs/#shepherd-team). Consider trying to raise interest by [posting in Discourse](https://discourse.nixos.org/), [taking in Matrix](https://matrix.to/#/#community:nixos.org) or reaching out to people that you know.
+>
+> If not enough shepherds can be found in the next month we will close this RFC until we can find enough interested participants. The PR can be reopened at any time if more shepherd nominations are made.
+>
+> [See more info on the Nix RFC process here](https://github.com/NixOS/rfcs/blob/master/README.md)
+
+If another month has elapsed since the warning without sufficent accepted nominations the following message will be posted, the `status: insufficent interest` label will be added and the issue will be closed.
+
+> This RFC is being closed due to lack interest. If enough shepherds are found this issue can be reopened. If you don't have permission to reopen please [open an issue for the NixOS RFC Steering Committee](https://github.com/NixOS/rfc-steering-committee/issues/new) linking to this PR.
+
+## Renewed Interest
+
+If an RFC that was closed due to lack of interest finds sufficent interest it can be reopened and the `status: insufficent interest` label removed. The RFC will then continue through the RFC process as normal.
```

Comment:
```suggestion
If an RFC that was closed due to lack of interest finds sufficient interest it can be reopened and the `status: insufficient interest` label removed. The RFC will then continue through the RFC process as normal.
```